### DISCUSSION TITLE
Databases on network drives not accessible anymore

### DISCUSSION THREAD

**Discussion by Sierron** [pawelsalawa/sqlitestudio#5170] [General]
### Details
Databases on a network drive can no longer be opened by the database dialog (add, edit). The reason is that they are not available in the open folder dialog.

### Steps to reproduce
Click on add or edit database.
Select existing database and open the folder dialog. Network drives do not appear in the list.
![database dialog](https://github.com/user-attachments/assets/5b32c715-3c98-4002-810b-7c17fe057d40)
![folder dialog](https://github.com/user-attachments/assets/c87d8cbe-14b0-4f6b-8189-669b84db6144)

### Operating system
Windows 10 64bit

### SQLiteStudio version
3.4.6

**Comment by pawelsalawa** [comment]
Seems to be working here:
![image](https://github.com/user-attachments/assets/a7d36509-a23d-463e-a843-a35ced2943e4)
Do you see your shared drive on the list when you execute `net use` in your `cmd`?

**Comment by Sierron** [comment]
Sorry for the long delay. I checked with the old version and made sure that it was working fine beforehand and checked that the network drives are visible in the file explorer. And they are. I've used the old version today just fine with files on a network drive.

After updating however I get the yellow warning icon on the databases that are on a network drive. The database type is removed and when I try to check the path of the database the network drives are not visible.

**Comment by Sierron** [comment]
> Seems to be working here: ![image](https://private-user-images.githubusercontent.com/481846/389266112-a7d36509-a23d-463e-a843-a35ced2943e4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQxOTMyMTAsIm5iZiI6MTczNDE5MjkxMCwicGF0aCI6Ii80ODE4NDYvMzg5MjY2MTEyLWE3ZDM2NTA5LWEyM2QtNDYzZS1hODQzLWEzNWNlZDI5NDNlNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNFQxNjE1MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMTRiZWJlZTEyYzM1NzIyYmUxOWRhMGVlM2YwNWI5MTM1NzE3ZDFjYzVjMDZkNTc1YzAwYjkyZmQwYzdkMTI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.X7Fev4gwKh008myRbBSKlua98vXoWEsij0ZWbasxlss) Do you see your shared drive on the list when you execute `net use` in your `cmd`?

And yes, running the command I can see the drives

**Comment by pawelsalawa** [comment]
It looks like your Windows is preventing SQLiteStudio from accessing this external drive.

Are you perhaps running this new version using some kind of compatibility emulation (that Windows built-in feature)? Or maybe you have a startup script that makes the SQLiteStudio run as different user? I don't know... but it doesn't seem to be issue with SQLiteStudio itself.

Have you tried running "portable" distribution of new SQLiteStudio version? (I assume you are currently using the "installer" based one?)

**Comment by Sierron** [comment]
So before I continue it's fixed now and I think I know why.

The installer needs to run as an admin and at the end there is a checked box saying open sqlitestudio. It looks like it opens it under admin user. And would you look at that:
![noaccess](https://github.com/user-attachments/assets/485f64a2-1c18-4434-a81a-cb5431848ab7)

I don't know why I didn't catch that earlier. Maybe I tried to open it as admin myself after the initial start with the installer and saw the same thing happening.

**Comment by pawelsalawa** [comment]
Good. By the way you should know, that using SQLite with shared/network files is not recommended. You may experience weird, unexpected results: https://www.sqlite.org/useovernet.html

**Comment by Sierron** [comment]
Thanks for the headsup and I am aware. Read that long time ago somewhere. My application isn't using network stored databases but rather I use sqlitestudio to edit these database files and the program/services run on the same machine as the databases. Unless SQLiteStudio is unreliable with that as well, there should be nothing to worry about for me.
